opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -25,54 +23,50 @@
 from opentelemetry.util._importlib_metadata import entry_points
 
 logger = logging.getLogger(__name__)
-_RUNTIME_CONTEXT = None  # type: typing.Optional[_RuntimeContext]
-_RUNTIME_CONTEXT_LOCK = threading.Lock()
 
-_F = typing.TypeVar("_F", bound=typing.Callable[..., typing.Any])
 
-
-def _load_runtime_context(func: _F) -> _F:
-    """A decorator used to initialize the global RuntimeContext
+def _load_runtime_context() -> typing.Optional[_RuntimeContext]:
```

Comment:
Now that we added fallback, this can longer be None

```suggestion
def _load_runtime_context() -> _RuntimeContext:
```